test/e2e/specs/editor/plugins/meta-boxes.spec.js

```diff
@@ -61,9 +61,17 @@ test.describe( 'Meta boxes', () => {
 		const postId = await editor.publishPost();
 		await page.goto( `/?p=${ postId }` );
 
-		await expect(
-			page.locator( '.wp-block-latest-posts > li' )
-		).toContainText( [ 'A published post', 'Dynamic block test' ] );
+		await expect
+			.poll( async () => {
+				const posts = await page
+					.locator(
+						'.entry-content .wp-block-latest-posts__post-title'
```

Comment:
I've updated the locator scope so it only checks the post content. The previous one could match the block in the widget area.